x/emissions/keeper/actor_utils/losses.go

```diff
@@ -22,17 +22,11 @@ import (
 func CloseReputerNonce(
 	k *keeper.Keeper,
 	ctx sdk.Context,
-	topicId keeper.TopicId,
+	topic types.Topic,
```

Comment:
why is the whole topic being passed in? not just topicId like before? no other part of the topic is being used but the id...

Reply:
we use the `AlphaRegret`, so we were redundantly querying it before...